.buildkite/scripts/steps/build_and_test.sh

```diff
@@ -32,14 +32,31 @@ if [[ -n "$BUILDKITE_PULL_REQUEST" && -n "$VERSION_QUALIFIER" ]] ; then
     exit 2
 fi
 
-# Tests must be run in PR builds
-if [[ -n "$BUILDKITE_PULL_REQUEST" && "$RUN_TESTS" = false ]] ; then
-    echo "RUN_TESTS should not be false PR builds"
-    exit 3
-fi
+VERSION=$(cat ${REPO_ROOT}/gradle.properties | grep '^elasticsearchVersion' | awk -F= '{ print $2 }' | xargs echo)
+HARDWARE_ARCH=$(uname -m | sed 's/arm64/aarch64/')
 
 echo "environment variables:"
 env
+
+if [[ "$HARDWARE_ARCH" = aarch64 && -z "$CPP_CROSS_COMPILE" ]] ; then 
+  # On Linux native aarch64 build using Docker
+  
+  # The Docker version is helpful to identify version-specific Docker bugs
+  docker --version
+  
+  KERNEL_VERSION=`uname -r`
+  GLIBC_VERSION=`ldconfig --version | head -1 | sed 's/ldconfig//'`
+
+  if [ "$RUN_TESTS" = false ] ; then
+    ${REPO_ROOT}/dev-tools/docker_build.sh linux_aarch64_native
+  else
+    ${REPO_ROOT}/dev-tools/docker_test.sh --extract-unit-tests linux_aarch64_native
+    echo "Re-running seccomp unit tests outside of Docker container - kernel: $KERNEL_VERSION glibc: $GLIBC_VERSION"
+    (cd ${REPO_ROOT}/cmake-build-docker/test/lib/seccomp/unittest && LD_LIBRARY_PATH=`cd ../../../../../build/distribution/platform/linux-aarch64/lib && pwd` ./ml_test_seccomp)
+  fi
+  exit $?
```

Comment:
I don't think it should exit here, as we want the artifact upload at the end.

But the `docker_entrypoint.sh` part below should only be called if running in Kubernetes.